protob/messages-monero.proto

```diff
@@ -0,0 +1,438 @@
+syntax = "proto2";
+
+// Sugar for easier handling in Java
+option java_package = "com.satoshilabs.trezor.lib.protobuf";
+option java_outer_classname = "TrezorMessageMonero";
+
+import "common.proto";
+
+
+/**
+ * Structure representing Monero public address
+ */
+message MoneroAccountPublicAddress {
```

Comment:
Let's drop the `m_` prefix.

Reply:
Yep why not. I took the field naming from the original Monero code but I don't like it either.